hs/src/LedgerState.hs

```diff
@@ -50,12 +49,7 @@ import           Delegation.StakePool    (Delegation (..), StakePool (..))
 -- | A ledger consists of a list of entries where each such entry is either a
 -- stake delegation step or a transaction.
 
-data LedgerEntry =
-    TransactionData !TxWits
-  | DelegationData !DCert
-    deriving (Show, Eq)
-
-type Ledger = [LedgerEntry]
+type Ledger = [TxWits]
```

Comment:
I don't think it is used anywhere.

Reply:
it was used in one test, but I've removed it and the alias now.